src/example.txt

```diff
@@ -0,0 +1 @@
+This is an example file
```

Comment:
😂 This file should either be removed completely or moved to a test folder if we use it in the tests.